modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2575,6 +2584,21 @@ private void _fillDefaultValue(
 
 			if (value != null) {
 				values.put(objectField.getName(), (Serializable)value);
+
+				if (!localized) {
+					continue;
+				}
+
+				if (localizedValues.isEmpty()) {
+					values.put(
+						objectField.getI18nObjectFieldName(),
+						HashMapBuilder.put(
+							defaultLanguageId, value
+						).build());
+				}
+				else {
+					localizedValues.putIfAbsent(defaultLanguageId, value);
+				}
```

Comment:
@pedro-oliveira446 can we use `computeIfAbsent` to avoid this `if/else` here?

```
Map<String, Object> localizedValues =
	(Map<String, Object>)values.computeIfAbsent(
		objectField.getI18nObjectFieldName(), key -> new HashMap<>());

...

localizedValues.putIfAbsent(defaultLanguageId, value);
```